state-transition/core/state_processor_forks.go

```diff
@@ -48,6 +48,7 @@ import (
 func (sp *StateProcessor) ProcessFork(
 	st *statedb.StateDB, timestamp math.U64, logUpgrade bool,
 ) error {
+	sp.logger.Debug("invoked ProcessFork") // a log needed for testing purposes
```

Comment:
current forks do not perform changes that are easily assertable from `simulated` tests. This little log simplifies the tests